ruby/command-t/scanner/file_scanner/find_file_scanner.rb

```diff
@@ -36,8 +36,9 @@ def paths!
             counter = 1
             next_progress = progress_reporter.update(counter)
             stdout.each_line do |line|
-              next if path_excluded?(line.chomp!)
-              paths << line[@prefix_len..-1]
+              line = line[@prefix_len..-2]
```

Comment:
One reason to prefer the `chomp!` by the way, is that it is (subjectively) more intention revealing: it essentially says "remove whitespace at the end, if present", while `-2` just says "I am a mysterious magic number". (Of course `-1` is still magical too.)